src/components/StrategyLiveChart/StrategyLiveChart.js

```diff
@@ -26,18 +25,11 @@ const StrategyLiveChart = ({
 
   return (
     <Panel
-      label=''
       removeable={false}
       moveable={false}
-      extraIcons={[
-        <span
-          key='toggle-fullscreen'
-          type='button'
-          className='icon-move toggle-fullscreen'
-          onClick={() => setFullScreenChart(!fullScreenChart)}
-          title='Toggle Fullscreen'
-        />,
-      ]}
```

Comment:
@dmytroshch please ignore this comment if chart integration is still in progress..
I see that chart full-screen button is moved, but onClick is not implemented in following code inside Options component, so chart full-screen won't work!

` <Button
        className='hfui-strategy-options__fullscreen-btn item'
        label='Fullscreen Chart'
        onClick={() => {}}
        green
      />`

Reply:
@bhoomij it still in progress